NitroxClient/MonoBehaviours/Gui/Chat/PlayerChat.cs

```diff
@@ -50,13 +50,14 @@ public IEnumerator SetupChatComponents()
             inputField.gameObject.AddComponent<PlayerChatInputField>().InputField = inputField;
             inputField.GetComponentInChildren<Button>().onClick.AddListener(playerChatManager.SendMessage);
 
-            backgroundImages = new[] { transform.GetChild(0).GetComponent<Image>(), transform.GetChild(1).GetComponent<Image>(), transform.GetChild(3).GetComponent<Image>() };
+            // We pick any image that's inside the chat component to have all of their opacity lowered
+            backgroundImages = transform.GetComponentsInChildren<Image>();
 
             yield return new WaitForEndOfFrame(); //Needed so Select() works on initialization
             IsReady = true;
             if (NitroxPrefs.SilenceChat.Value)
             {
-                Log.InGame(Language.main.Get("Nitrox_SilencedChatNotif"));
+                Log.InGame(Language.main.Get("Nitrox_SilencedChatNotify"));
```

Comment:
As we delay the key change after weblate merge this should also be reverted

Reply:
Ah yes well found